ssv/spectest/tests/runner/consensus/invalid_decided_value.go

```diff
@@ -39,19 +40,32 @@ func InvalidDecidedValue() tests.SpecTest {
 				Runner: testingutils.SyncCommitteeContributionRunner(ks),
 				Duty:   &testingutils.TestingSyncCommitteeContributionDuty,
 				Messages: []*types.SSVMessage{
-					testingutils.SSVMsgSyncCommitteeContribution(nil, testingutils.PreConsensusContributionProofMsg(ks.Shares[1], ks.Shares[1], 1, 1)),
-					testingutils.SSVMsgSyncCommitteeContribution(nil, testingutils.PreConsensusContributionProofMsg(ks.Shares[2], ks.Shares[2], 2, 2)),
-					testingutils.SSVMsgSyncCommitteeContribution(nil, testingutils.PreConsensusContributionProofMsg(ks.Shares[3], ks.Shares[3], 3, 3)),
+					testingutils.SSVMsgSyncCommitteeContribution(
+						nil,
+						testingutils.PreConsensusContributionProofMsg(ks.Shares[1], ks.Shares[1], 1, 1),
+					),
+					testingutils.SSVMsgSyncCommitteeContribution(
+						nil,
+						testingutils.PreConsensusContributionProofMsg(ks.Shares[2], ks.Shares[2], 2, 2),
+					),
+					testingutils.SSVMsgSyncCommitteeContribution(
+						nil,
+						testingutils.PreConsensusContributionProofMsg(ks.Shares[3], ks.Shares[3], 3, 3),
+					),
 
 					testingutils.SSVMsgSyncCommitteeContribution(
-						testingutils.TestingCommitMultiSignerMessageWithIdentifierAndFullData(
-							[]*bls.SecretKey{ks.Shares[1], ks.Shares[2], ks.Shares[3]},
+						testingutils.TestingCommitMultiSignerMessageWithHeightIdentifierAndFullData(
+							[]*bls.SecretKey{
+								ks.Shares[1], ks.Shares[2], ks.Shares[3],
+							},
 							[]types.OperatorID{1, 2, 3},
+							qbft.Height(testingutils.TestingDutySlot),
 							testingutils.SyncCommitteeContributionMsgID,
 							consensusDataByts(types.BNRoleSyncCommitteeContribution),
-						), nil),
+						), nil,
+					),
 				},
-				PostDutyRunnerStateRoot: "0e966c2d067b51f57254f5e56acd98673f026c89f3d4df74bd567f9668417501",
+				PostDutyRunnerStateRoot: "aff4af0dbbead81d6cb9dd4ff734d4660712a8b5ab8e9016a3f86e2c2ead7549",
```

Comment:
if we don't change it will fail? it takes the post root from the json no?

Reply:
It needs to be correct, because of backwards compatability check you added.
Also note the json take the value from the code...